app/controllers/workflows_controller.rb

```diff
@@ -128,6 +128,12 @@ def destroy
   def submit
     set_workflow
 
+    # We want to allow the user to resubmit a job that has been run or failed. This will clear old jobs from
```

Comment:
Maybe clarify that "this will delete preexisting job records for this workflow" in the comment. Instead of just saying "clear old jobs from a workflow". 

Also, is there a reason why you choose clear over delete_all over destroy_all? Might want to mention that in the comments too.